astroquery/ipac/irsa/tests/test_irsa_remote.py

```diff
@@ -87,3 +87,10 @@ def test_tap(self):
             result = Irsa.query_tap(query=query)
         assert len(result) == 5
         assert result.to_table().colnames == ['ra', 'dec']
+
+    def test_ssa(self):
+        coord = SkyCoord.from_name("Eta Carina")
+        result = Irsa.query_ssa(pos=coord)
+        assert len(result) > 260
+        collections = set(unique(result.to_table(), keys='dataid_collection')['dataid_collection'])
```

Comment:
is `set(unique(` redundant?  I'd imagine so

Reply:
sure 🤦‍♀️